src/frontend/web_application/src/components/MessageList/components/Message/presenter.jsx

```diff
@@ -147,7 +151,7 @@ class Message extends Component {
             )}
             {message.date_insert &&
```

Comment:
the condition here is wrong when `isMessageFromUser` is true (and can broke the app if date is undefined). May be this can be done in a `renderDate` method.